examples/parcel/.compiledcssrc.json

```diff
@@ -11,6 +11,5 @@
     ]
   ],
   "extract": true,
-  "optimizeCss": false,
-  "classNameCompressionMapFilePath": "./class-name-compression-map.json"
```

Comment:
not sure if this still serves a purpose - probably more confusing to keep it in when no products are actually using this

it also breaks the pseudo-selector ordering and @kylorhall-atlassian's increaseSpecificity stuff

Reply:
cc @liamqma ^^^

I'll need to remind myself to update `examples/webpack` too